integrations/ad_reader.py

```diff
@@ -158,6 +142,7 @@ def read_user(self, user=None, cpr=None):
             self.results[current_user['SamAccountName']] = current_user
 
         response = self._get_from_ad(user=user, cpr=cpr, school=False)
+        current_user = None
         for current_user in response:
             job_title = current_user.get('Title')
```

Comment:
Does current_user need to be both in and out of the scope.
If yes, i think it would be less confusing to use 2 different identifiers, e.g. current_user and user_in_loop (or similar).

Reply:
Dette er default værdien hvis iterationen aldrig kører, den var så dog forkert og burde have været {}